util/emailer.py

```diff
@@ -25,15 +25,16 @@ def send(self):
         msg.attach(MIMEText(self.body, "plain"))
 
         for attachment in self.attachments:
-            with open(attachment, "rb") as file:
-                aFile = MIMEApplication(file.read(), _subtype="txt")
-                aFile.add_header("Content-Disposition", "attachment",
-                    filename=basename(file))
-                msg.attach(file)
+            split = attachment.split('/')
+            name = split[-1]
+            file = MIMEApplication(open(attachment, 'rb').read(),
+                                   _subtype="txt")
+            file.add_header("Content-Disposition", "attachment", filename=name)
```

Comment:
This file is never closed. Will that cause memory issues?

Reply:
From my testing it doesn't seem to cause any memory leaks or anything, but I'll update this PR later today.

Reply:
And Resolved!

Reply:
Perfect!